### PR TITLE
Support for Imodane, the Pyrohammer

### PR DESCRIPTION
**Pull request by kevlahnota**

### CODE REVIEW

File: forge-game/src/main/java/forge/game/trigger/TriggerDamageDoneOnce.java [hunk 1]
```diff
@@ -36,6 +37,21 @@ public boolean performTest(Map<AbilityKey, Object> runParams) {
             return false;
         }
 
+        if (hasParam("SingleTarget")) {
+            Object target = runParams.get(AbilityKey.DamageTarget);
+            final Map<Card, Integer> damageMap = (Map<Card, Integer>) runParams.get(AbilityKey.DamageMap);
+            for (Card source : getDamageSources(damageMap)) {
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/trigger/TriggerDamageDoneOnce.java:43]: if you use DamageDone trigger instead you wouldn't need to iterate over all sources?

File: forge-game/src/main/java/forge/game/trigger/TriggerDamageDoneOnce.java [hunk 2]
```diff
@@ -36,6 +37,21 @@ public boolean performTest(Map<AbilityKey, Object> runParams) {
             return false;
         }
 
+        if (hasParam("SingleTarget")) {
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/trigger/TriggerDamageDoneOnce.java:40]: clearer name like "SourceDamagedTarget"?

**Comment by Hanmac** [comment]
i got a different aproach, going to commit soon, but i'm taking the card script

**Comment by Hanmac** [comment]
i think in most cases, the castSA should work?

i mean is there a card that can cause an Instant and Sorcery to deal damage (while not being the Spell itself?)

**Comment by kevlahnota** [comment]
closes in favor of #3759

**Pull request closed without merging**